src/google_cloud_debugger/google_cloud_debugger_lib/dbg_stack_frame.cc

```diff
@@ -125,6 +112,21 @@ HRESULT DbgStackFrame::Initialize(
     return hr;
   }
 
+  if (!is_async_method_) {
```

Comment:
Even is we are in an async method will there be none async variables?  Consider the case:

```
var temp = 3;
Task.Run(() => 
{
... 
})
```

Reply:
I believe if you are using Visual Studio, you can only see variable temp inside the `Task.Run` block if you reference it in the block. Otherwise you can't. We have similar behavior except we will show `temp` in a `this` variable. I'll improve this behavior in the next few PR.